api/src/pcapi/core/users/api.py

```diff
@@ -171,6 +172,9 @@ def create_account(
             user.externalIds = {}
         user.externalIds["apps_flyer"] = {"user": apps_flyer_user_id, "platform": apps_flyer_platform.upper()}  # type: ignore [index, call-overload]
 
+    if firebase_pseudo_id:
+        user.externalIds["firebase_pseudo_id"] = firebase_pseudo_id  # type: ignore [index, call-overload]
+
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**🎯 suggestion:** ‏Tu dois avoir besoin de vérifier comme au dessus si on a `user.externalIds` et l'initialiser à `{}` si non.

Je sortirais ça des `if` même:

Reply:
Après vérification, `externalIds` vaut `{}` par défaut. 
Donc **🎯 suggestion:** ‏l'inverse, retirer 
```python 
    if user.externalIds is None:
            user.externalIds = {}

Reply:
En cherchant sur Internet, j'ai trouvé que `default` est appliqué au moment de l'INSERT/UPDATE, je vais remettre la condition du coup